ah/kubernetes_clusters.go

```diff
@@ -50,6 +50,11 @@ type ClustersAPI interface {
 	Update(context.Context, string, *ClusterUpdateRequest) error
 	GetConfig(context.Context, string) (string, error)
 	Delete(context.Context, string) error
+	GetNodePool(context.Context, string, string) (*KubernetesNodePool, error)
```

Comment:
minor: consider using `Kubernetes` prefix everywhere, so `GetNodePool` would become `GetKubernetesNodePool` and the same about the methods below